package.json

```diff
@@ -4,7 +4,9 @@
   "description": "API prototype: the peer that knows all things",
   "main": "index.js",
   "scripts": {
```

Comment:
We can write

```
      "bin": {
        "mediachain-cli": "npm run build >/dev/null && node ./lib/client/cli/index.js"
      },
```

instead of the entry in scripts?

(also I'm assuming we'll remove the `build` before shipping)